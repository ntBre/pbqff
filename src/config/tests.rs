use insta::assert_debug_snapshot;

use super::*;

#[test]
fn config() {
    assert_debug_snapshot!(Config::load("testfiles/test.toml"));
}

#[test]
fn path_template() {
    assert_debug_snapshot!(Config::load("testfiles/path.toml"));
}

#[test]
fn normal() {
    assert_debug_snapshot!(Config::load("testfiles/normal.toml"));
}
