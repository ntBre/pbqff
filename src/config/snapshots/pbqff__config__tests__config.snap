---
source: src/config/tests.rs
expression: "Config::load(\"testfiles/test.toml\")"
snapshot_kind: text
---
Config {
    geometry: Zmat(
        "C\nC 1 CC\nC 1 CC 2 CCC\nH 2 CH 1 HCC 3 180.0\nH 3 CH 1 HCC 2 180.0\n\nCC =                  1.42101898\nCCC =                55.60133141\nCH =                  1.07692776\nHCC =               147.81488230\n",
    ),
    optimize: true,
    charge: 0,
    step_size: 0.005,
    coord_type: Sic,
    template: "scfcrt=1.D-21 aux(precision=14 comp xp xs xw) PM6 THREADS=1",
    hybrid_template: "scfcrt=1.D-21 aux(precision=14 comp xp xs xw) PM6 THREADS=1",
    queue_template: None,
    program: Mopac,
    queue: Slurm,
    sleep_int: 2,
    job_limit: 2048,
    chunk_size: 1,
    findiff: false,
    check_int: 100,
    weights: None,
    dummy_atoms: None,
    norm_resume_hff: false,
}
